public-api/db-cdr/generate-cdr/init-new-cdr-db.sh

```diff
@@ -32,8 +34,8 @@ fi
 # export for liquibase to use this
 #export CDR_DB_NAME
 
-# If CDR_DB_NAME matches ^public, we want to the public_db_user env var substituted in the create_db.sql
-if [[ CDR_DB_NAME =~ ^public ]]
+# If version flag contains public, we want to the public_db_user env var substituted in the create_db.sql
+if [[ $VERSION_FLAG =~ .*public.* ]]
```

Comment:
Alternatively, could you remove this if check and the flags entirely? The CDR codepath should never be used in this repo.